cou/utils/openstack_lookup.csv

```diff
@@ -18,7 +18,7 @@ swift,2.24.0,2.26.0,2.26.0,2.27.0,2.27.0,2.28.0,2.28.0,2.29.0,2.29.0,2.30.0
 ceph,15.2.0,16.2.0,15.2.0,16.2.0,16.2.0,17.2.0,16.2.0,17.2.0,17.2.0,18.2.0
 gnocchi,4.3.0,4.4.0,4.4.0,4.4.1,4.4.0,4.4.1,4.4.1,4.4.2,4.4.1,4.4.2
 designate-bind,9.16.1,9.18.12,9.16.1,9.18.12,9.16.1,9.18.12,9.16.1,9.18.12,9.16.1,9.18.12
-ovn,22.03,22.09,22.03,22.09,22.03,22.09,22.03,22.09,22.03,22.09
+ovn,20.03,22.09,20.03,22.09,20.03,22.09,20.03,22.09,20.03,22.09

```

Comment:
As discussed elsewhere, we will confirm the lower bound for each release instead of putting all of them as `20.03`. From what we are seeing [here](https://docs.openstack.org/charm-guide/latest/project/procedures/ovn-upgrade-2203.html#updated-neutron-api-plugin-ovn-charm), it seems, at least for yoga, it's lower bound should be 22.03

Reply:
correct, 22.03 is what's in the jammy repos, 20.03 is in the focal repos